src/main/java/com/fvv/bookstore/util/DbPropertiesUtil.java

```diff
@@ -0,0 +1,70 @@
+package com.fvv.bookstore.util;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Load a DataBase properties file to connection factory.
+ * 
+ * @author Fatima Vasquez
+ * <p>Created on 2018</p>
+ * @version 1.0 	
+ *
+ */
+
+public class DbPropertiesUtil {
+	
+	private static Properties prop;	
+	private static final String DB_DRIVER = "db.mysql.driver";
+	private static final String DB_URL = "db.mysql.url";
+	private static final String DB_USER = "db.mysql.user";
+	private static final String DB_PASS = "db.mysql.pass";
+	
+	static {
+		prop = new Properties();
+		
+		try (InputStream input = new FileInputStream(Constants.CONFIG_DB_PROPERTIES_NAME)) {
+			prop.load(input);
+		} catch (IOException e) {
+			System.err.println("Error: " + e);
+		}
+	}
+	
+	/**
+	 * It reads a property value from a key stored in a file with a key-value format.
```

Comment:
you used the same javadoc from the previous class, you MUST adjust this javadoc to mention what this specific method does.